task04/Main.java

```diff
@@ -0,0 +1,94 @@
+package com.homework;
+import java.lang.*;
+import java.util.*;
+
+/**
+ * The program calculates real roots of quadratic equation. Parameters of
+ * quadratic equation got from the keyboard and shows to the command line
+ */
+public class Main {
+    /**
+     * Method calculateEquation gets array of parameters, verifies
+     * that discriminant isn't negative. The method returns array
+     * of equation roots
+     * @param params array of the parameters of quadratic equation
+     * @return array of equation roots
+     */
+    public static double[] calculateEquation (double[] params) {
+        double[] result = new double[2];
+        // Calculates discriminant of the quadratic equation
+        double discriminant = params[1] * params[1] - 4 * params[0] * params[2];
+        //Verifies and performs actions in case of negative and null discriminant
+        if (Double.compare(discriminant, 0) < 0) {
+            System.out.println("Negative discriminant. Program calculates only real roots");
+            return result;
```

Comment:
Actually this is not correct your method returns result[0] =0 and result[1] = 0, caller cannot check if it's two roots 0 and 0 or it signals about any issue. Yes, you print a message for user, but not for piece of code that calls your method. 
Suggestion: throw an exception here